test/handler.test.js

```diff
@@ -199,9 +200,31 @@ describe('strong-error-handler', function() {
       requestJson().end(function(err, res) {
         if (err) return done(err);
 
-        var expectedData = {statusCode: 500, stack: error.stack};
-        for (var key in error) expectedData[key] = error[key];
+        var expectedData = {
+          statusCode: 500,
+          message: 'a test error message',
+          name: 'ErrorWithProps',
+          details: 'some details',
+          extra: 'sensitive data',
+          stack: error.stack,
+        };
+        expect(res.body).to.have.property('error');
+        expect(res.body.error).to.eql(expectedData);
+        done();
+      });
+    });
 
+    it('contains name property when debug=true', function(done) {
```

Comment:
This is already tested by `contains all error properties when debug=true`, isn't it? I am proposing to remove this test. Unless there is something special about the test setup that I don't see?